data/json/items/melee/bludgeons.json

```
@@ -477,6 +477,45 @@
     "price": 1600,
     "price_postapoc": 10
   },
+  {
+    "type": "GENERIC",
+    "id": "club_wooden",
+    "name": { "str": "wooden club" },
+    "description": "This is little more than a plank of wood, snapped in half and with a crude handle whittled into one end.  It looks similar to a poorly-made cricket bat.",
+    "weight": "1200 g",
+    "price_postapoc": 10,
+    "color": "brown",
+    "symbol": "/",
+    "looks_like": "cudgel",
+    "material": [ "wood" ],
+    "techniques": [ "WBLOCK_1" ],
+    "weapon_category": [ "MACES" ],
```

This seems most like a heavy baton to me.
```suggestion
    "weapon_category": [ "BATONS" ],
```

---

strong disagree; the thing is still a 2x4 in dimensions beyond the handle. batons are narrower objects.

---

Oh, I thought this was more narrow, but you are right, I'm not convinced "MACES" is the most appropiate category for this weapon (Maybe it shouldn't have any, but well, I did not make the Barbaran weapon categories, so I could be wrong), I will mark this conversation as resolved then.

---

i can!